place_resolver.py

```diff
@@ -76,6 +77,58 @@ def filter_place_entries(self, entries):
 
         return place_entries
 
+    def process_length_3_coords(coords):
+        d = coords[0]
+        m = coords[1]
+        se = coords[2]
+        dd = 0;
+        if d == 0:
+            dd = ((m / 60.0) + (se / 3600.0));
+        else:
+            # python doesn't have a clean way to get the symbol of a number so we use copysign
+            dd = math.copysign(1,d) * abs(d) + (m / 60.0) + (se / 3600.0)
+        return round(dd, 6)
+
+    def process_length_2_coords(coords):
+        d = coords[0]
+        m = coords[1]
+        return round(math.copysign(1,d) * abs(d) + (m / 60.0), 6)
+
+    def clean_coordinates(coord):
+        split_string =  coord.split("|")
+        coords = []
+        formatted_coords = []
+        for string in split_string:
+            try:
+                # check if string is a int/float and cast to float
+                coords.append(float(string))
+            except:
+                pass
+            if string in ["N", "E"]:
```

Comment:
I take it that N, E, S, W are at the end of a coordinate string? an example in a comment/documentation would help to understand what this is doing.